### PR TITLE
Fix withdraw guidance

### PR DESCRIPTION
**Pull request by benthorner**
This fixes a few issues with the withdraw feature:

   - Requirements issues should not be persistent flashes
   - The guidance should be confined to the central page
   - The README should correctly say how to enable the feature

### CODE REVIEW

**Comment by benthorner** [comment]
@alex-ju I did a quick fix for the font size of the code example in the guidance, to stop it spilling outside the newly constrained guidance width. Is there a better way to approach this?

File: app/views/withdraw/new.html.erb [hunk 1]
```diff
@@ -24,33 +26,34 @@
...
-        <%= render "govuk_publishing_components/components/button", {
-          text: "Withdraw document", margin_bottom: true
-        } %>
+    <div class="govuk-grid-column-one-third">
+      <%= render "components/contextual_guidance", {
+        id: "withdrawal-public-explanation-guidance",
+        title: t("withdraw.new.public_explanation.guidance_title"),
+      } do %>
+        <p>Explain to users why the page has been withdrawn. You can link to alternative content with markdown.</p>
+        <pre class="app-c-markdown-guidance__code-snippet" style="font-size: 12px">
```
Comment (alex-ju) [inline app/views/withdraw/new.html.erb]: Thanks for flagging this, I missed it when reviewing.
I would recommend adding `word-break: break-word;` to the `.app-c-markdown-guidance__code-snippet` class to deal with text overflowing issues and preserve the font-size as is.

**Comment by benthorner** [comment]
@alex-ju thanks :-). I've pushed a fix - OK to merge?

**Comment by alex-ju** [comment]
@benthorner. Yep, looks good now.

**Pull request merged**